securedrop_proxy/proxy.py

```diff
@@ -128,11 +232,14 @@ def handle_response(self):
         else:
             self.handle_non_json_response()
 
+        # https://mypy.readthedocs.io/en/latest/kinds_of_types.html#union-types
+        # To make sure that mypy knows the type of self.res is not None.
+        assert self.res
```

Comment:
these asserts are OK because they will be handled by the generic exception handling below (`self.simple_error` will be called)